### PR TITLE
Stepper

### LINKED ISSUE #178: Stepper
- Create Quest
- Schedule claim

### PR DESCRIPTION
**Pull request by kafann**
close #178

### CODE REVIEW

**Comment by Corantin** [comment]
Finalement je pensse sa serait plus lisible si c'etait la meme taille

File: packages/react-app/src/components/modals/quest-modal.tsx [hunk 1]
```diff
@@ -128,18 +123,18 @@ export default function QuestModal({
...
-          <QuestActionButtonStyled
-            key="btn-save"
-            label="Create"
-            mode="positive"
-            type="submit"
-            form={`form-quest-form-${data?.address ?? 'new'}`}
-          />
-        ),
-      ]}
+      // buttons={[
```
Comment (Corantin) [inline packages/react-app/src/components/modals/quest-modal.tsx]: Remove comments

File: packages/react-app/src/components/modals/schedule-claim-modal.tsx [hunk 2]
```diff
@@ -242,75 +246,96 @@ export default function ScheduleClaimModal({
         onSubmit={(values, { setSubmitting }) => {
           onClaimSubmit(values, setSubmitting);
         }}
-        validateOnBlur
+        // validateOnBlur
```
Comment (Corantin) [inline packages/react-app/src/components/modals/schedule-claim-modal.tsx]: Remove comments

**Review by Corantin — CHANGES_REQUESTED** [review]
Tu pourra merge apres sa, jai fait pas mal de truc dans ta PR mb haha jai add sa :
- Wallet balance dans quest
- Disable Create quest button when form invalid
- Le reste jpensse jte lai dit

**Review by Corantin — APPROVED** [review]

**Review by Corantin — APPROVED** [review]
To Validate : 
- [x] Stepper in create Quest modal
- [x] Stepper in Schedule Claim modal
- [x] Wallet balance is now in create Quest modal and should show a loading feedback
- [ ] Claim list in Quest detail should show loading as well as no content feedback
- [ ] Amount field input should show loading feedback (in placeholder)

**Comment by alexchan0394** [comment]
**Wallet balance is now in create Quest modal and should show a loading feedback** 

When creating a quest, unselecting a token will show loading of the wallet balance when it should be "No Token Selected" 

- Create quest
- Select a token
- Deselect token
- Click BACK 
- Click NEXT
- You should see no token selected with a loading wallet balance

image

**Comment by Corantin** [comment]
> **Wallet balance is now in create Quest modal and should show a loading feedback**
> 
> When creating a quest, unselecting a token will show loading of the wallet balance when it should be "No Token Selected"

Solved by [fce349a](https://github.com/1Hive/quests/pull/235/commits/fce349ad9f52535eae019ead11a9e93727cf319b)

**Comment by alexchan0394** [comment]
> To Validate :
> 
> * [x]  Stepper in create Quest modal
> * [x]  Stepper in Schedule Claim modal
> * [x]  Wallet balance is now in create Quest modal and should show a loading feedback
> * [x]  Claim list in Quest detail should show loading as well as no content feedback
> * [x]  Amount field input should show loading feedback (in placeholder)

I checked everything and it should be good to go

**Pull request merged**